_data/sites.json

```
@@ -4605,6 +4605,17 @@
         ]
     },
 
+    {
+        "name": "MeetFrank",
+        "url": "https://meetfrank.com",
+        "email": "team@meetfrank.com",
+        "difficulty": "easy",
```

```suggestion
        "difficulty": "hard",
```
I know sending an e-mail is easy, but we deem the deletion hard because it requires someone on the service side to process your e-mail, which might be fast, slow or never happen.

---

I got a reply within minutes, but fair enough.